packages/sdk/package.json

```diff
@@ -74,20 +92,30 @@
         "clean": "rimraf -- lib src/grpc-api"
     },
     "dependencies": {
+        "@azure/core-asynciterator-polyfill": "^1.0.2",
         "@concordium/rust-bindings": "2.0.0",
         "@grpc/grpc-js": "^1.9.4",
         "@noble/ed25519": "^2.0.0",
+        "@noble/hashes": "^1.3.2",
         "@protobuf-ts/grpc-transport": "^2.9.1",
         "@protobuf-ts/grpcweb-transport": "^2.9.1",
         "@protobuf-ts/runtime-rpc": "^2.8.2",
         "@scure/bip39": "^1.2.1",
+        "@stardazed/streams-polyfill": "^2.4.0",
         "@types/big.js": "^6.2.0",
         "big.js": "^6.2.0",
         "bs58check": "^3.0.1",
         "buffer": "^6.0.3",
         "hash.js": "^1.1.7",
         "iso-3166-1": "^2.1.1",
         "json-bigint": "^1.0.0",
+        "react-native-polyfill-globals": "^3.1.0",
```

Comment:
**Suggestion:** That is a lot of new dependencies just needed for one platform. I would suggest we instruct the user how to polyfill their environment instead of making it part of the library.

Reply:
Removed them

Reply:
They seem to still be there though. Am I missing something?